source/unsafe-messenger.js

```diff
@@ -73,6 +74,34 @@ export default function unsafeMessenger() {
 			editor.blur();
 		});
 	}
+
+	function monacoEditor(target) {
+		const data_uri = target.getAttribute("data-uri");
+		const editor = monaco.editor.getModels(data_uri)[0];
+		let currentValue = editor.getValue();
+		sendBack(target, currentValue);
+		const throttledSend = throttle(50, sendBack);
+
+		editor.onDidChangeContent((e) => {
```

Comment:
If it works, it's best to wrap this instead

```suggestion
		editor.onDidChangeContent(throttle(event => {
```

Also it should be named `event`, not just `e`. Refer to the linter’s suggestions